tv.sh

```diff
@@ -8,34 +8,38 @@ TITLES=(
   "SIC"
   "TVI"
   "RTP 3"
-  "SIC Noticias"
+  "SIC Notícias"
   "CNN Portugal"
-  "RTP Memoria"
+  "RTP Memória"
   "RTP Madeira"
-  "RTP Acores"
-  "RTP Africa"
+  "RTP Açores"
+  "RTP África"
   "RTP Internacional"
-  "SIC Internacional"
+  "TVI Internacional"
+  "TVI Ficção"
+  "TVI Reality"
   "ARTV"
   "Porto Canal"
   "Euronews"
   "Kuriakos TV"
 )
 
 STREAMS=(
-  "__rtp rtp1"
-  "__rtp rtp2"
+  "https://streaming-live-app.rtp.pt/liverepeater/smil:rtp1HD.smil/rtp_playlist.m3u8"
+  "https://streaming-live-app.rtp.pt/liverepeater/smil:rtp2HD.smil/rtp_playlist.m3u8"
   "https://sic.live.impresa.pt/sic.m3u8"
   "__tvi"
-  "__rtp rtp3"
+  "https://streaming-live-app.rtp.pt/liverepeater/smil:rtpnHD.smil/rtp_playlist.m3u8"
```

Comment:
This change doesn't work for me, but the old __rtp url is working for rtp3

Reply:
If you tried manually you must put an user-agent.

For example on mpv:

`mpv https://streaming-live-app.rtp.pt/liverepeater/smil:rtpnHD.smil/rtp_playlist.m3u8 --http-header-fields='User-Agent: RTP Play'`

If you use the TV.sh it will automatically create a random user-agent for you and open the stream immediately.